### ISSUE TITLE
Remove the transparent invisible square in SVGs

### ISSUE THREAD

**Issue by huayunh** [etn-ccis/blui-icons#197]
#### Describe the desired behavior

Remove all the invisible squares from icons in the `svg` folder. People using PowerPoint cannot import icons into PPT and change color due to this extra square element "`<path fill="none" d="M0 0h24v24H0z"/>`" (or its equivalent)

#### Describe the current behavior

It was there because React Native had some specific requirements for icon buttons.

#### Is this request related to a current issue?

MarComm proposed in a MarComm discussion

**Comment by huayunh** [comment]
See file changes from https://github.com/brightlayer-ui/icons/pull/88 for a headstart

**Comment by bkarambe** [comment]
@huayunh Any updates on this one?

**Comment by huayunh** [comment]
No, this issue is still valid. What updates are you looking for, @bkarambe

**Comment by bkarambe** [comment]
This issue still exists.

**Comment by huayunh** [comment]
someone else from Studio Blue is having trouble using certain BLUI icons because of this issue
